recipes/scherbelberg/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "scherbelberg" %}
```

Comment:
Love the name by the way. :)

Reply:
Thanks :) Have you read the [project's name section](https://github.com/pleiszenburg/scherbelberg#projects-name) in the readme?